scss/ionic.app.scss

```diff
@@ -40,5 +40,7 @@ $ionicons-font-path: "../lib/ionic/release/fonts" !default;
 @import "scss/pages/accounts";
 @import "scss/pages/help";
 @import "scss/pages/realitycheck";
+@import "scss/pages/profittable";
+@import "scss/pages/realaccountopening";
```

Comment:
Is this for real account opening only? why not call it "newaccount" and include all virtual/real?